reana_job_controller/kubernetes_job_manager.py

```diff
@@ -191,3 +201,42 @@ def add_shared_volume(self, job):
         job['spec']['template']['spec']['containers'][0][
             'volumeMounts'].append(volume_mount)
         job['spec']['template']['spec']['volumes'].append(volume)
+
+    def _add_krb5_sidecar_container(self, job, secrets_volume_mount):
+        """Add  sidecar container for a job."""
+        krb5_config_map_name = 'krb5-conf'
```

Comment:
Let's move this to config and state very well that this name depends on the one [set on REANA-Cluster/backends/kubernetes/templates/configmaps/krb5.yaml](https://github.com/reanahub/reana-cluster/pull/224/files#diff-41bb518496363149415e1666d76ece21R5).